.github/workflows/deploy.yml

```diff
@@ -23,10 +23,16 @@ jobs:
         uses: actions/checkout@v3
         with:
           persist-credentials: false
+      - uses: awalsh128/cache-apt-pkgs-action@v1
+        with:
+          packages: calibre
+          version: 1.0
```

Comment:
Is it needed to specify a version? Reading through the source I see this is the cache version, not the calibre version. I'd suggest to drop it and only add it once we need it. You can also purge caches via the UI nowadays so it's not needed as much anymore.